frontend/src/LineGraph.res

```diff
@@ -14,7 +14,6 @@ type global
 @module("dygraphs")
 external global: global = "default"
 
-%%raw(`import 'dygraphs/src/extras/synchronizer.js'`)
```

Comment:
This was producing an error. Was that import needed for the graph synchronization feature?

![image](https://user-images.githubusercontent.com/5595092/108965641-6efd0f80-767d-11eb-8b51-5b7db369aa2c.png)

Reply:
Interesting. This is exactly the error I had when I tried using esbuild. I think dygraphs is doing something fancy there.

I'm happy to drop the synchronization functionality for now. It's not essential.